### PR TITLE
[eclipse/xtext#1770] update to mwe 2.11.3 final

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1770] update to mwe 2.11.3 final
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
TODO: adapt feature timestamp once mwe release is done

File: releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<unit id="org.eclipse.xtend.lib.source" version="0.0.0"/>
 		<unit id="org.eclipse.xtend.lib.macro" version="0.0.0"/>
 		<unit id="org.eclipse.xtend.lib.macro.source" version="0.0.0"/>
-		<repository location="https://ci.eclipse.org/xtext/job/xtext-lib/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+		<repository location="https://ci.eclipse.org/xtext/job/xtext-lib/job/cd_mwe2113final/lastStableBuild/artifact/build/p2-repository/"/>
```
Comment (szarnekow) [inline releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target:12]: Obvious, but better safe than sorry: Don't forget to revert this change.
